samples/matter/template/boards/nrf54l15dk_nrf54l15_cpuapp_internal.conf

```diff
@@ -10,7 +10,6 @@ CONFIG_BT_LL_SOFTDEVICE_MULTIROLE=y
 # nRF54L15 requires bigger stack sizes than nRF52/nRF53 families
 CONFIG_CHIP_MALLOC_SYS_HEAP_SIZE=10240
 CONFIG_MPSL_WORK_STACK_SIZE=2048
-CONFIG_CHIP_TASK_STACK_SIZE=7168
```

Comment:
is the workaround not needed for `samples/matter/template/boards/nrf54l15dk_nrf54l15_cpuapp_internal.conf`?

Reply:
Yes, thanks for the catch.